.github/workflows/osl_sync_upstream.yml

```diff
@@ -2,28 +2,43 @@ name: "CI :: OSL :: Sync Upstream"
 
 on:
   workflow_dispatch: # Allows you to run the workflow manually from the Actions tab
+    inputs:
+      dry_run:
+        description: "True to perform a dry run (dry run git push and skip create PR step)"
+        required: false
+        default: "false"
+
   schedule:
     - cron: "0 2 * * 1" # Runs every Monday at 2 AM UTC (adjust as needed)
 
 env:
-  UPSTREAM_URL: https://github.com/apache/incubator-kie-tools
+  UPSTREAM_URL: ${{ secrets.UPSTREAM_URL }}
```

Comment:
@ricardozanini wouldn't it be better to have it in the `vars` instead of the `secrets` as it is useful to read in the logs?

Reply:
I've changed that already, no?

Reply:
Ops, no, I've changed in the other PR.